### PR TITLE
pocket_lab: '_available' attribute fixes and adapting to recent lvgl changes

### PR DESCRIPTION
**Pull request by mphalke**
1. Removed dependency on the attribute location/index from GUI dropdown list to read/write attributes, as this creates issues reading available attributes. available attributes are not stored into GUI dropdown list, so indexing them results into reading of some other attribute and not the intended one. All attributes are now read/write by passing there name from GUI layer and then finding the matching name into attribute params structure into IIO wrapper layer
2. Updated lvgl function calls and variables related to object event set, keypad button matrix and lvgl image array as lvgl library commit is bumped up to latest lvgl library version (master branch). This is most likely going to be next stable version of lvgl library and hence changes are needed. Again, in order to support latest changes from vendor lvgl port code (stm32f769ni_disco_lvgl) the master branch verison of lvgl library needs to be used

### CODE REVIEW

File: pocket_lab/pl_gui_iio_wrapper.c [hunk 1]
```diff
@@ -270,12 +270,12 @@ int32_t pl_gui_scan_global_attr_avail_options(const char *attr_name,
...
  * @return	0 in case of success, negative error code otherwise
  */
-int32_t pl_gui_scan_chn_attr_avail_options(const char *attr_name,
+int32_t pl_gui_get_chn_attr_avail_options(const char *attr_name,
 		char *attr_val, uint32_t chn_indx, uint32_t dev_indx)
 {
 	struct iio_device *iio_dev;
 	struct iio_ch_info chn_info;
-	char buf[50];
+	char buf[100];
```
Comment (ribdp) [inline pocket_lab/pl_gui_iio_wrapper.c:278]: Was this change intentional?

Comment (ribdp) [reply pocket_lab/pl_gui_iio_wrapper.c:278]: Nvm, it probably makes sense to have a larger buffer to accommodate all the options for an attr.

**Review by ribdp — APPROVED** [review]
Looks good to me

**Review by CMinajigi — APPROVED** [review]

File: pocket_lab/pl_gui_iio_wrapper.c (hunk 1 above)
Comment (mphalke) [reply pocket_lab/pl_gui_iio_wrapper.c:278]: ya, that was the reason the size is increased
